src/main/resources/application.yml

```diff
@@ -27,6 +27,7 @@ management:
     web:
       exposure:
         include: info
+imageDirectory: kimonoimages
```

Comment:
When we look at where it's being used, perhaps "imageUploadPath" could be a more suitable name instead of "imageDirectory"